lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1650,9 +1672,16 @@ def cmd_sessions(*args)
         end
       end
     when 'killall'
-      print_status("Killing all sessions...")
-      framework.sessions.each_sorted.reverse_each do |s|
-        session = framework.sessions.get(s)
+      if matching_sessions
+        print_status("Killing matching sessions...")
+        print_line
+        print(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```

Comment:
should this be a `print_status`?
```suggestion
        print_status(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```

Reply:
Was just going with the existing functionality, but that's no reason not to give it a swap if we think it's better

Reply:
I think this question became automagically unresolved; It shouldn't be `print_status` here as the table would be printed with a `[*] ` prefix which isn't needed